rules/no-unused-properties.js

```diff
@@ -139,7 +139,7 @@ const create = context => {
 							return {identifier: parent};
 						}
 
-						return null;
+						return undefined;
```

Comment:
@fisker Seems like we have an idea for another rule here. `no-return-undefined`. Since `return` and `return` is the same thing.

Reply:
See this https://github.com/sindresorhus/eslint-plugin-unicorn/pull/636#issuecomment-611326069

Reply:
I'm not talking about this rule. I'm talking in general about `return undefined` being used (anywhere).

Reply:
`no-useless-undefined`

- `return undefined;`
- `let a = undefined;`
- `call(undefined)`
- more?

Reply:
```js
const {foo = undefined} = {};

function ({foo = undefined} = {}) {}

function (foo = undefined) {}
```